apps/auth/Dockerfile

```
@@ -1,20 +1,35 @@
-RUN bun run build
+RUN --mount=type=cache,target=/root/.bun \
+    bun run build
 
-FROM oven/bun:latest AS runtime
-WORKDIR /app
-RUN apt-get update -y && apt-get install -y openssl
-COPY --from=base /app/node_modules ./node_modules
-COPY --from=base /app/packages ./packages
-COPY --from=base /app/apps/auth ./apps/auth
+FROM node:22-alpine AS runtime
+RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)